pkg/tool/az-analyze/cmd/azv.go

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
Let's move the sources to `cmd/az-analyze` rather than put it under `'pkg/tool`.